tree/dataframe/src/RDataFrame.cxx

```diff
@@ -690,7 +704,10 @@ namespace ROOT {
 /// See RInterface for the documentation of the
 /// methods available.
 RDataFrame::RDataFrame(std::string_view treeName, TDirectory *dirPtr, const ColumnNames_t &defaultBranches)
-   : RInterface<RDFDetail::RLoopManager>(std::make_shared<RDFDetail::RLoopManager>(nullptr, defaultBranches))
+   : RInterface<RDFDetail::RLoopManager>(
+        std::make_shared<RDFDetail::RLoopManager>(nullptr, defaultBranches),
+        RDFInternal::RBookedCustomColumns(std::make_shared<RCustomColumnBasePtrMap_t>(),
+                                          std::make_shared<ColumnNames_t>()))
```

Comment:
here and below, can `RDFInternal::RBookedCustomColumns(std::make_shared<RCustomColumnBasePtrMap_t>(), std::make_shared<ColumnNames_t>())` be replaced by `{}`?
If not, can it be replaced by `RDFInternal::RBookedCustomColumns({}, {})`?

Reply:
@bluehood It gives a segfault because it is not able to initialize the structures properly. I am adding a costructor to hide this initialization.
```
////////////////////////////////////////////////////////////////////////////
/// \brief Creates a new wrapper with empty maps
RBookedCustomColumns():
   fCustomColumns(std::make_shared<RCustomColumnBasePtrMap_t>()), fCustomColumnsNames(std::make_shared<ColumnNames_t>())
   {
   }
```

Reply:
I bet it gives a segfault because of null pointers..

Reply:
That's right

Reply:
Changed in 189adf5